event_attribution_reporting_clicks.md

```diff
@@ -250,13 +248,7 @@ delete the scheduled report with the lowest priority and schedule the new report
 
 ### Multiple sources for the same trigger (Multi-touch)
 
-If multiple sources were clicked and associated with a single attribution trigger, send reports for all of them. 
-
-To provide additional utility, the browser can choose to provide additional annotations to each of these reports, attributing credits for the triggering redirect to them individually. Attribution models allow for more sophisticated, accurate measurement.
-
-The default attribution model will be last-click attribution, giving the last-clicked source for a given trigger redirect all of the credit.
-
-To remain flexible, the browser sends an `credit` of value 0 to 100 for all reports associated with a single trigger. This represents the percent of attribution a source received. The sum of credits across a set of reports for one trigger should equal 100.
+If multiple sources were clicked and associated with a single attribution trigger, send reports for the one with the highest priority.
```

Comment:
source priority isn't a concept in this explainer.

Should we upstream attributionsourceprioirty in this as well?

If not this should read, send a report for the most recent source if we wanted to be self-consistent in the explainer. But adding in priority makes the most sense imo, as it is already in the spec.

Reply:
Done.